tezos/new_context/src/ffi.rs

```diff
@@ -649,6 +649,14 @@ ocaml_export! {
         timings::context_action(rt, action_name, key, irmin_time, tezedge_time);
         OCaml::unit()
     }
+
+    fn tezedge_timing_init(
+        rt,
+        db_path: OCamlRef<String>,
+    ) {
+        timings::init_timing(rt, db_path);
```

Comment:
Convert the string here instead and pass it directly, that way you don't have to pass the runtime handle around.